### DISCUSSION TITLE
table cell() 存在时 ,是否 dataIndex 其实可以不要?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#43526] [Q&A]
Table 的 dataIndex:[a,b]
和 cell: (text,.row) => row.a.b 是不是一个效果?

当 dataSource 是很复杂的嵌套对象时, 你们文档写的是使用 dataIndex : string[], 
想问一下 直接使用 cell 函数 是不是也可以? 然后 dataIndex 空着,这样会有其他的问题吗?

**Comment by linxianxi** [comment]
看文档里 Column key 属性的说明

**Reply by vaynevayne** [reply]
<img width="988" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/f89d9711-e2ed-469f-bfe1-12c81c89b270">
<img width="965" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/eec8b8dc-ec5e-4fda-86f2-573b182cc383">

结论是 可以不要 or 必须要? 这也没说啊

**Comment by afc163** [comment]
是一个效果，dataIndex 是个快捷的语法糖。

**Reply by vaynevayne** [reply]
多谢
